example/DSSAT/original/PETPT_DSSAT.f

```diff
@@ -0,0 +1,315 @@
+C=======================================================================
+
+C  PETPT, Subroutine, J.T. Ritchie
```

Comment:
It seems to me that the code is duplicated. Isn't it?

Reply:
Yes, it is. I correct it.